JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/JavaModificationExpressionVisitor.java

```diff
@@ -70,6 +96,26 @@ public Void visitCompare(CompareExpression expression) {
 
 	@Override
 	public Void visitCall(CallExpression expression) {
+
+		// ToDo: Scenario: We have a Getter and Setter on the type, how do we do a postfix call here?
+		//         Currently this is reached for every instance field since we always generate an auto getter and setter
+		if(expression.method.method instanceof GetterMember) {
```

Comment:
🏳️ Help: This should _really_ not be done at this part of the compilation (especially since this wouldn't work if the getter was part of the class but the setter part of an expansion).

Where in the compilation chain does this make sense?